cleanup/cleanup.sh

```diff
@@ -71,9 +71,9 @@ get_deletion_candidates() {
     local inactive_istags
     local deletion_candidates
 
-    commit_list=($(get_commit_list))
-    inactive_istags=($(get_inactive_imagestreamtags))
-    deletion_candidates=()
+    commit_list="($(get_commit_list))"
+    inactive_istags="($(get_inactive_imagestreamtags))"
+    deletion_candidates="()"
```

Comment:
Are you sure this works? I'm worried we just get strings with this.

I believe, `shellcheck` [didn't complain](https://travis-ci.com/appuio/oc-plugins/builds/90854024), did it?

Reply:
Tested on an an actual cluster, worked as expected (with **no** matches).

What might work better is replacing `local` with `declare -a`. It does the same thing (declare a local variable) but explicitly set it's type to be an "indexed array variable".